.circleci/config.yml

```diff
@@ -1,30 +1,118 @@
-version: 2
+version: 2.1
+executors:
+  integration_test_exec:
+    docker: # run the steps with Docker
+      - image: cimg/openjdk:11.0.8
+        environment:
+          PGHOST: 127.0.0.1
+      - image: circleci/postgres:11.6
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: postgres
+          PG_HOST: localhost
+          POSTGRES_HOST_AUTH_METHOD: trust
+          POSTGRES_PASSWORD: postgres
+      - image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
+        environment:
+          - xpack.security.enabled: false
```

Comment:
Does this always work? I guess so... I found out the hard way that I couldn't run on my Mac unless I added 
```
   ulimits:
     memlock:
      soft: -1
      hard: -1
```
But maybe it's because I'm on a Mac.

Reply:
seems to work. I think Travis just uses the default